lms/envs/labster.py

```diff
@@ -25,6 +25,7 @@
     'labster_course_license',
     'labster_vouchers',
     'labster_enroll',
+    'labster_accounts',
```

Comment:
Does it make sense to move it private repo. For eampke,  `labster_course_license` and rename this application later as a part of separate PR?